tests/scripts/pulp_ostree/test_import_publish.sh

```diff
@@ -15,7 +15,7 @@ tar --exclude="index.html" -cvf "fixtures_small_repo.tar" -C "$OSTREE_DOWNLOADED
 
 expect_succ pulp ostree repository create --name "cli_test_ostree_repository"
 
-if pulp debug has-plugin --name "ostree" --min-version "2.0.0a6.dev"
+if pulp debug has-plugin --name "ostree" --min-version "2.0.0.dev"
```

Comment:
@mdellweg, I think the current implementation in pulp-glue is not working as intended. I had to change the minimum requirements all over the place because of the nightly build.

```
>>> from packaging.version import parse
>>> parse("2.0.0.dev") > parse("2.0.0a6.dev")
False
```

I would assume the above returns `True`.

Reply:
Also, you may notice there is pulp_ostree 2.0.0.dev installed instead of pulp_ostree 2.1.0.dev. This should be resolved tomorrow because I merged this commit just today and the nightly image was not re-built: https://github.com/pulp/pulp_ostree/commit/20f4ed2a5fed8aaa01271947820cb8b94f1d2f6f

Reply:
After some discussion, we have decided it is worth using released versions, like "2.0.0" instead of "2.0.0.dev" versions.

```
>>> parse("2.0.0") > parse("2.0.0a6")
True

```